packages/strapi-design-system/src/Field/FieldLabel.tsx

```diff
@@ -29,17 +29,28 @@ export const FieldLabel = forwardRef<HTMLLabelElement, FieldLabelProps>(
     }
 
     return (
-      <Typography ref={ref} variant="pi" textColor="neutral800" htmlFor={id} fontWeight="bold" as="label" {...props}>
-        <Flex alignItems="center">
-          {children}
-          {required && <TypographyAsterisk textColor="danger600">*</TypographyAsterisk>}
-          {action && <Action marginLeft={1}>{action}</Action>}
-        </Flex>
-      </Typography>
+      <TypographyFlex
+        ref={ref}
+        variant="pi"
+        textColor="neutral800"
+        htmlFor={id}
+        fontWeight="bold"
+        as="label"
+        {...props}
+      >
+        {children}
+        {required && <TypographyAsterisk textColor="danger600">*</TypographyAsterisk>}
+        {action && <Action marginLeft={1}>{action}</Action>}
+      </TypographyFlex>
     );
   },
 );
 
+const TypographyFlex = styled(Typography)`
+  display: flex;
+  align-items: center;
+`;
```

Comment:
I changed it because it wasn't correctly being picked up in the tests and it just adds to dom nesting for the sake of it 🤷🏼‍♀️  which imo is probably more harmful then adding the css?

Reply:
So the issue was that the text returned, when you query for the label, was not properly returned?

Reply:
Something like that, it definitely couldn't find it, because it was in a div not a span, I then felt it didn't really need to be two levels deep for the sake of not making another class for it.

Reply:
Ok, I think I can agree to keep it :) Would you mind adding a comment to explain this an optimization?